pkgs/by-name/di/dive/package.nix

```diff
@@ -31,26 +30,6 @@ buildGoModule rec {
     lvm2
   ];
 
-  patches = [
-    # fix scrolling
-    # See https://github.com/wagoodman/dive/pull/447
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Oh right, I should have noticed that! Amended in 77cdf04a794f